installer/manifests/keptn/charts/control-plane/templates/ingress.yaml

```
@@ -1,5 +1,9 @@
 {{- if .Values.ingress.enabled -}}
+{{- if lt .Capabilities.KubeVersion.Minor "19" }}
+apiVersion: networking.k8s.io/v1beta
+{{- else }}
 apiVersion: networking.k8s.io/v1
+{{- end -}}
```

```suggestion
{{- end }}
```
If both dashes are there, a newline will be missing in the rendered chart